packages/contractkit/src/wrappers/Governance.ts

```diff
@@ -72,4 +118,369 @@ export class GovernanceWrapper extends BaseWrapper<Governance> {
+      this.kit,
+      this.contract.methods.vote(parseNumber(proposalID), proposalIndex, vote)
+    )
+  }
+
+  /**
+   * Executes a given proposal's associated transactions.
+   * @param proposalID Governance proposal UUID
+   */
+  async execute(proposalID: NumberLike) {
+    const proposalIndex = await this.getDequeueIndex(proposalID)
+    return toTransactionObject(
+      this.kit,
+      this.contract.methods.execute(parseNumber(proposalID), proposalIndex)
+    )
+  }
+
+  /**
+   * Returns approved, executed, and prepared status associated with a given hotfix.
+   * @param hash keccak256 hash of hotfix's associated abi encoded transactions
+   */
+  getHotfixRecord = proxyCall(
+    this.contract.methods.getHotfixRecord,
+    tupleParser(parseBuffer),
+    (o) => ({
+      approved: o[0],
+      executed: o[1],
+      preparedEpoch: toBigNumber(o[2]),
+    })
+  )
+
+  /**
+   * Returns whether a given hotfix has been whitelisted by a given address.
+   * @param hash keccak256 hash of hotfix's associated abi encoded transactions
+   * @param whitelister address of whitelister
+   */
+  isHotfixWhitelistedBy = proxyCall(
+    this.contract.methods.isHotfixWhitelistedBy,
+    tupleParser(parseBuffer, (s: Address) => identity<Address>(s))
+  )
+
+  /**
+   * Marks the given hotfix whitelisted by `sender`.
+   * @param hash keccak256 hash of hotfix's associated abi encoded transactions
+   */
+  whitelistHotfix: (hash: Buffer) => CeloTransactionObject<void> = proxySend(
+    this.kit,
+    this.contract.methods.whitelistHotfix,
+    tupleParser(parseBuffer)
+  )
+
+  /**
+   * Marks the given hotfix approved by `sender`.
+   * @param hash keccak256 hash of hotfix's associated abi encoded transactions
+   * @notice Only the `approver` address will succeed in sending this transaction
+   */
+  approveHotfix: (hash: Buffer) => CeloTransactionObject<void> = proxySend(
+    this.kit,
+    this.contract.methods.approveHotfix,
+    tupleParser(parseBuffer)
+  )
+
+  /**
+   * Marks the given hotfix prepared for current epoch if quorum of validators have whitelisted it.
+   * @param hash keccak256 hash of hotfix's associated abi encoded transactions
+   */
+  prepareHotfix: (hash: Buffer) => CeloTransactionObject<void> = proxySend(
+    this.kit,
+    this.contract.methods.prepareHotfix,
+    tupleParser(parseBuffer)
+  )
+
+  /**
+   * Executes a given sequence of transactions if the corresponding hash is prepared and approved.
+   * @param transactions Sequence of transactions
+   * @notice keccak256 hash of abi encoded transactions computed on-chain
+   */
+  executeHotfix(transactions: Transaction[]) {
+    const enc = encodedTransactions(transactions)
+    return toTransactionObject(
+      this.kit,
+      this.contract.methods.executeHotfix(enc.values, enc.destinations, enc.data, enc.dataLengths)
+    )
+  }
+}
+
+export function encodedTransactions(transactions: Transaction[]) {
+  if (transactions.length === 0) {
```

Comment:
this doesn't seem to be a check that belong to this fuction, but to the propose() method